connection-api/src/main/java/org/terracotta/exception/ConnectionClosedException.java

```diff
@@ -44,7 +46,18 @@ public ConnectionClosedException(String description) {
    * @param description The description of the exception.
    * @param cause The underlying cause of the exception.
    */
-  public ConnectionClosedException(String description, Throwable cause) {
+  public ConnectionClosedException(boolean wasSent, String description, Throwable cause) {
     super(null, null, description, cause);
+    this.wasSent = wasSent;
+  }
+  
+  /**
+   * Designates whether the message associated with this exception was sent to the cluster.
+   * 
+   * @return true of the message was sent to the server
+   */
+  
+  public boolean messageWasSent() {
```

Comment:
The sense of this seems wrong. Definite action should only be taken here when this method returns false.  Essentially the return values here are really: `false` and `maybe`. I'm wondering if an inverted method might lead to less mistakes?

Reply:
makes sense. will fix